spec/views/events.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe "events/index", type: :view do
+  context "events page" do
+    it "lists events ordered by start asc in localtime" do
+      assign(:events, [
+               event1 = FactoryBot.create(:event),
+               event2 = FactoryBot.create(:event, start: "2019-09-08 03:00:00", finish: "2019-09-08 20:00:00", error: nil, success: true, created_at: "2019-09-08 20:00:00", updated_at: "2019-09-08 20:00:00"),
+               event3 = FactoryBot.create(:event, start: "2019-11-07 03:00:00", finish: "2019-11-07 20:00:00", error: nil, success: true, created_at: "2019-11-07 20:00:00", updated_at: "2019-11-07 20:00:00")
```

Comment:
Maybe we should use TimeCop to create these events in the past?

Reply:
I will update them with Time.now +- 1.month or 2.month etc